### ISSUE TITLE
Fail to deploy

### ISSUE THREAD

**Issue by 88866j** [ServiceWeaver/weaver-gke#53]
Hello, tried a few times to deploy example app but got the following errors

```
Enabling multi-cluster ingress for cluster "serviceweaver-config" in "us-central1"... Error
Enabling multi-cluster ingress for cluster "serviceweaver-config" in "us-central1"... Error
Stopping with error: ERROR: (gcloud.container.fleet.ingress.enable) INVALID_ARGUMENT: InvalidValueError for field config_membership: Membership "projects/sw-test-collatz/locations/global/memberships/serviceweaver-config-us-central1" does not exist
: exit status 1
ERROR: (gcloud.container.fleet.ingress.enable) INVALID_ARGUMENT: InvalidValueError for field config_membership: Membership "projects/sw-test-collatz/locations/global/memberships/serviceweaver-config-us-central1" does not exist
: exit status 1
exit status 1
```

gcloud version:
```
Google Cloud SDK 428.0.0
alpha 2023.04.25
beta 2023.04.25
bq 2.0.91
core 2023.04.25
gcloud-crc32c 1.0.0
gke-gcloud-auth-plugin 0.5.2
gsutil 5.23
```

weaver gke version
```
weaver gke: commit=? deployer=v0.2.0 target=darwin/arm64
```

gcloud container fleet memberships list
```
NAME                              EXTERNAL_ID                           LOCATION
serviceweaver-config-us-central1  292d4399-3120-4197-b299-ccb5092ea921  us-central1
```

gcloud container fleet memberships describe --location global serviceweaver-config-us-central1
```
ERROR: (gcloud.container.fleet.memberships.describe) NOT_FOUND: Resource 'projects/sw-test-collatz/locations/global/memberships/serviceweaver-config-us-central1' was not found
- '@type': type.googleapis.com/google.rpc.ResourceInfo
  resourceName: projects/sw-test-collatz/locations/global/memberships/serviceweaver-config-us-central1
```

**Comment by spetrovic77** [comment]
Thank you for filing the bug, looks like this is a common issue.

One weird thing is that in your case, the cluster fleet membership gets restricted to `us-central1` location. In our cases, we get `global`:

```
NAME                                                  EXTERNAL_ID                                                   LOCATION
serviceweaver-config-us-central1  64a640fe-e473-4dff-9793-724ad21224e5  global
```

I'll try to understand what's going on. We definitely want the `global` membership.

**Comment by spetrovic77** [comment]
This should be fixed now as of weaver-gke@v0.7.1.

Please note that you may have to update your weaver dependency to @v0.7.1 as well.
